src/firebase.ts

```diff
@@ -0,0 +1,62 @@
+import { initializeApp } from 'firebase/app'
+import {
+  deleteDoc,
+  doc,
+  DocumentData,
+  getDoc,
+  getFirestore,
+  setDoc
+} from 'firebase/firestore'
+import {
+  ImageMessage,
+  TextMessage,
+  User
+} from './types'
+
+const firebaseConfig = {
+  apiKey: 'AIzaSyCaYDFqWQvrA9nYlg446x0XKHTejrWJea0',
```

Comment:
`apiKey` and other sensitive data should be taken from .env:
https://www.npmjs.com/package/dotenv